website/content/docs/commands/auth/move.mdx

```diff
@@ -0,0 +1,34 @@
+---
+layout: docs
+page_title: auth move - Command
+description: |-
+  The "auth move" command moves an existing auth method to a new path. Any
+  tokens from the old auth method are revoked, but all configuration
+  associated with the engine is preserved.
+---
+
+# auth move
+
+The `auth move` command moves an existing auth method to a new path. Any
+leases from the old auth method are revoked, but all configuration associated
+with the engine is preserved. The command can be issued for a move within or across
+namespaces, using namespace prefixes in the arguments.
+
+The command will trigger a remount operation, and use the returned migration ID to poll the 
+status of the operation, until a terminal state of `success` or `failure` is reached.
+
+**Moving an existing auth method will revoke any leases from the old
+method.**
+
+## Examples
+
+Move the existing auth method at ns1/approle/ to ns2/new-approle/:
+
+```shell-session
+$ vault secrets move ns1/approle/ ns2/new-approle/
```

Comment:
Should this be vault auth move?

Reply:
yep!

Reply:
[ec185fc](https://github.com/hashicorp/vault/pull/14314/commits/ec185fca4801fb73794fecd7dda77eea8a8e6d22)